tools/bin/cassandra.in.sh

```diff
@@ -119,7 +114,11 @@ esac
 
 # Read user-defined JVM options from jvm-server.options file
 JVM_OPTS_FILE=$CASSANDRA_CONF/jvm${jvmoptions_variant:--clients}.options
-if [ $JAVA_VERSION -ge 11 ] ; then
+if [ $JAVA_VERSION -ge 22 ] ; then
```

Comment:
same comment as above - we should just allow exact versions

Reply:
Same response as above :D